charters/Architecture/charter.md

```diff
@@ -0,0 +1,35 @@
+# Architecture++
+
+## Description
+
+Architecture++ is responsible for the technical oversight of the Data Coordination Platform (DCP), including identifying and addressing *cross-cutting* architectural concerns across DCP components.
+
+The core team is composed of *Technical Leads* representing each DCP component to ensure a diversity of skills and views needed for community technical decision-making. 
+
+## In-scope
+Strategic responsibilities
+* Set overall **technical** direction for the DCP
+* Publish an annual Architecture RFC documenting technical debt that needs to be addressed during roadmap planning by DCP PM++
+* Review and approve technical charters for DCP components
+* Review and approve cross-component RFC(s)
+    * Available to mentor on all RFC designs
+* Develop technical standards and best practices including, but not limited to:
+    * Technology Radar
+    * Ways of Working 
+    * Guidelines for DevSecOps policies
+    * Code review
+    
+Tactical responsibilities
+* Guide integration and deployment of DCP components *as a whole*
+* Coordinate emerging tactical request with DCP PM++ that need to be addressed during sprint planning by DCP PM++
```

Comment:
How about:
_Coordinate with DCP PM on emerging tactical requests needing to be addressed in Sprint planning priorities_